### PR TITLE
Add a quick start section

### PR DESCRIPTION
**Pull request by rminnich**
Each chapter should have one where possible.

I am not sure the command is right.

### CODE REVIEW

File: 10.u-root/README.md [hunk 1]
```diff
@@ -4,6 +4,45 @@ Points of contact: [Ronald Minnich](https://github.com/rminnich)
...
+## Quick Start
+
+Build a u-root initramfs in busybox mode.
+
+	$ go run github.com/u-root/u-root -build=bb
+	2019/02/11 18:29:00 Build environment: GOARCH=amd64 GOOS=linux GOROOT=/home/rminnich/go GOPATH=/home/rminnich/gopath CGO_ENABLED=0
+	2019/02/11 18:29:00 Filename is /tmp/initramfs.linux_amd64.cpio
+	2019/02/11 18:29:12 Successfully wrote initramfs.
+
+This command scans all the u-root source code with the GO AST package, converting commands into packages, and compiling them into one binary. It takes around fifteen seconds.  It creates a file called /tmp/initramfs.linux_amd64.cpio. To test it, you need to install the testramfs command.
```
Comment (insomniacslk) [inline 10.u-root/README.md]: nit: you could briefly specify that this process saves a large amount of space, proportional to the number of programs

File: 10.u-root/README.md [hunk 2]
```diff
@@ -4,6 +4,45 @@ Points of contact: [Ronald Minnich](https://github.com/rminnich)
...
+You can then run it. 
+
+	$ sudo $GOPATH/bin/testramfs -i /tmp/initramfs.linux_amd64.cpio
+	... lots of messages
+	 Created mount -t "cgroup" -o perf_event "cgroup" "/sys/fs/cgroup/perf_event" flags 0x0
+	 2019/02/11 18:30:31 Done Rootfs
+	 open /home/rminnich/.elvish/rc.elv: no such file or directory
+	 (unreachable)/home/rminnich/projects/linuxboot/book/10.u-root#    
+
+This will drop you into an interactive session. You can try again with dynamic compilation:
```
Comment (insomniacslk) [inline 10.u-root/README.md]: dynamic is source-mode, right? Worth specifying `-build=source`, or do you prefer people to figure out that it's the default?

File: 10.u-root/README.md (hunk 1 above)
Comment (rminnich) [reply 10.u-root/README.md]: done

File: 10.u-root/README.md (hunk 2 above)
Comment (rminnich) [reply 10.u-root/README.md]: done. I added a note.

**Pull request merged**